### ISSUE TITLE
Provide an example using seed = 1821

### ISSUE THREAD

**Issue by wsamuelw** [cutterkom/generativeart#7]
Could you please share the code to generate the art using the seed 1821?

**Comment by cutterkom** [comment]
Hey, 

the example is provided in the Readme. 

In the table at the bottom I provide the formulas for `x` and `y` for seed 1821. When you plug them into a variable you can run the function, see https://github.com/cutterkom/generativeart#examples

**Comment by wsamuelw** [comment]
Thanks. My goal is to re-generate the same image using the seed 1821 to learn more about the generate_img() function and the package in general. I get how to specify the formula X & Y. But, how to set seed to generate one single image with the above seed? Thanks again

**Comment by cutterkom** [comment]
I updated the package, because due to your issue I realized that the `regenerate_img()` was broken. 

So, your have to do two things: 
1. create a csv logfile just like at the bottom of the "Examples" section: https://github.com/cutterkom/generativeart#examples and reference it with `LOGFILE_PATH`
2. then call `regenerate_img(seed=1821, polar=TRUE)`. This will pull the formula from the logfile and recreate the image
